### ISSUE TITLE
Override doesn't work with property

### ISSUE THREAD

**Issue by NeilGirdhar** [microsoft/pyright#4877]
```python
from typing_extensions import override

class X:
    @property
    @override  # Should be an error!
    def f(self) -> int:
        raise NotImplementedError

class Y(X):
    @property
    @override  # Should be an error!
    def h(self) -> int:
        return 0
```

**Comment by erictraut** [comment]
PEP 698 does mention `property` (which I missed in my initial reading), but it doesn't provide much guidance about how that might work. In particular, properties can involve multiple methods — a getter, setter, and deleter. I chose to implement the `@override` check only on the method that is decorated with the `@property` decorator. The others (typically the setter and deleter) come along for the ride.

This will be included in the next release.

**Comment by NeilGirdhar** [comment]
Awesome!   That's perfect for me; thank you!

It's crazy how many corner cases there are :smile:

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.302, which I just published. It will also be included in a future release of pylance.
